### ISSUE TITLE
Source code?

### ISSUE THREAD

**Issue by AndreLobato** [teosusnjak/covid19NZ#1]
Hey buddy, awesome work! Would you have or want to add the source code for generating the plots and reports?

**Comment by teosusnjak** [comment]
Yeah, I've been thinking of putting some portions of it up. I need to be careful because some of the data to do with the economic analysis is confidential. So when I get a chance to tidy things up, I'll put it up.
